__tests__/integrations/Session/AuthenticationUser.test.js

```diff
@@ -23,10 +23,11 @@ describe("Authentication User", () => {
       password,
     });
 
-    expect(response.statusCode).toBe(200);
-    expect(response.body.user.name).toBe(name);
-    expect(response.body.user.email).toBe(email);
-    expect(response.body.token).toBeTruthy();
+    expect(response.statusCode).toBe(201);
```

Comment:
I think the correct status for authentication is 200. At least it is what I see most used. I've never seen 201 ...

Reply:
estava 200, mas o código antigo estava como 201...

Reply:
Right, fixed!